test/activate_set.jl

```diff
@@ -39,4 +39,7 @@
             Pkg.activate(orig_project_toml_path)
         end
     end
+
+    # https://github.com/JuliaTesting/TestEnv.jl/issues/26
+    @test isdefined(TestEnv, :isfixed)
```

Comment:
Welp, I guess that works.
Feels unsatisfying but, sure why not.

Reply:
Completely agree. Presumably the fact that it isn't hit in the existing tests means that a chunk of code doesn't get tested :(